helix-term/src/ui/editor.rs

```diff
@@ -124,7 +124,15 @@ impl EditorView {
             Box::new(highlights)
         };
 
-        Self::render_text_highlights(doc, view.offset, inner, surface, theme, highlights);
+        Self::render_text_highlights(
+            doc,
+            view.offset,
+            inner,
+            surface,
+            theme,
+            highlights,
+            &editor.config.whitespace,
```

Comment:
Should we just store this within editor?

Reply:
Yeah I think there's a method or two in Editor that end up using the config. Seems wise to make that available so we don't need to pass it in all the time.